CONTRIBUTING.md

```diff
@@ -10,10 +10,13 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 We welcome contributions to OSConfig. The main way of contributing to and extending OSConfig is via developing new **Management Modules**. For more information see [OSConfig Management Modules](docs/modules.md).
 Pull requests with few exceptions must contain appropriate unit-tests. We cannot allow test coverage to go down. Pull requests containing code changes without accompanying unit tests may be rejected.
-Pull requests needs to be formatted according to .pre-commit-config.yaml. Each pull request is checked by [Forrmating Tests](https://github.com/Azure/azure-osconfig/blob/dev/.github/workflows/formatting.yml).
+
 
 Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com. When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repositories using our CLA.
 
+# Coding style
+The coding style is described in a separate [style document](STYLE.md).
```

Comment:
>[style document](STYLE.md). [](http://example.com/codeflow?start=44&length=27)

Can you please move this under docs rather than keep it in root of the project (also renamed in lowercase as "codingstyle.md)? Thanks